internal/interfaces/grpc/http_utils.go

```diff
@@ -334,3 +331,32 @@ func isValidGrpcWebOptionRequest(req *http.Request) bool {
 		strings.Contains(accessControlHeader, "x-grpc-web") &&
 		strings.Contains(accessControlHeader, "content-type")
 }
+
+func isHttpRequest(req *http.Request) bool {
+	return strings.ToLower(req.Method) == "get" ||
```

Comment:
we should check also for `post` methods.

Reply:
Done

Reply:
mmm gRPC calls are post calls. also we always check for content type json OR get in other proj

Reply:
i think this was fine

Reply:
Done